frontend/src/lib/components/commandpalette.svelte

```diff
@@ -242,7 +247,7 @@ SPDX-License-Identifier: MPL-2.0
 				</p>
 				<input
 					type="text"
-					autofocus
+                    comment="svelte recommends against autofocus for accessibility reasons. Therefore removed autofocus command for this input"
```

Comment:
Yea, but at this point, the console is so full of errors and that doesn't matter here and in my opinion, when you open the command palette, you should be able to start typing immediately without clicking on the text field. Other websites do that as well. (Github, etc)

Reply:
Okay, I see that. It would be more convenient to be able to start typing automatically. I also want to implement the ability to tab between edit boxes (if that sounds good?) 

I will look more into how best to deal with the A11y warnings.

Reply:
Sounds awesome, but let's try to keep things with one at a time and for now, let's focus on getting Pydantic V2 and the new ormar working.